test/otel/testdata/ocb_build_script.sh

```diff
@@ -4,7 +4,19 @@ OTELCOL_PID=0
 
 mkdir -p /tmp/otel-ci
 trap 'rm -rf /tmp/otel-ci && kill $OTELCOL_PID' EXIT
+current_dir=$(pwd)
+replaces="/tmp/otel-ci/replaces"
+# Get path of all datadog modules, in sorted order, without the initial dot
+dd_mods=$(find . -type f -name "go.mod" -exec dirname {} \; | sort | sed 's/.//')
+echo "replaces:" >> "$replaces"
```

Comment:
In my version of the script I had this intermediary `replaces` file because I potentially had multiple `ocb` builds to do, but here you can probably pipe the replace statements directly into `/tmp/otel-ci/builder-config.yaml`.